src/doc_builder/build_doc.py

```diff
@@ -438,6 +438,10 @@ def check_toc_integrity(doc_folder, output_dir):
     # We don't just loop directly in toc as we will add more into it as we un-nest things.
     while len(toc) > 0:
         part = toc.pop(0)
+        if "local" in part:
```

Comment:
Is this change related to the `package_name` arg - I don't quite follow the connection / reason behind this logic

Reply:
thats a great suggestion! added in https://github.com/huggingface/doc-builder/pull/213/commits/2a2e9ad27000b6c4b1de7df2fc1ad553c1d0dca2